modules/appeals_api/spec/models/notice_of_disagreement_spec.rb

```diff
@@ -86,6 +86,17 @@
     end
   end
 
+  context 'birth date isn\'t in the past' do
```

Comment:
I agree with @ElleeB - let's match this wording with the text you added for claimant: "veteran birth date is in the future"